src/rasterstats/main.py

```diff
@@ -16,6 +16,8 @@
     remap_categories,
 )
 
+from tqdm import tqdm
```

Comment:
This import needs to be made optional.

```suggestion
try:
    from tqdm import tqdm
except ImportError:
    tqdm = None
```